packages/office-ui-fabric-react/src/components/Link/Link.types.ts

```diff
@@ -13,7 +13,7 @@ export interface ILink {
   focus(): void;
 }
 
-export interface ILinkProps extends React.AllHTMLAttributes<HTMLAnchorElement | HTMLButtonElement | HTMLElement | LinkBase> {
+export interface ILinkProps extends React.AnchorHTMLAttributes<HTMLAnchorElement | HTMLButtonElement | HTMLElement | LinkBase> {
```

Comment:
Hmm. I think AnchorHTMLAttributes is less specific than All. Wouldn't this be a breaking change?